tests/natasha_homework_7.py

```diff
@@ -0,0 +1,91 @@
+import time
+import unittest
+
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.select import Select
+
+from tests import CHROME_PATH
+
+
+class Homework_7(unittest.TestCase):
+    def setUp(self) -> None:
+        # browser session
+        browser = webdriver.Chrome(executable_path=CHROME_PATH)
+        browser.get('http://hrm-online.portnov.com/')
+        self.browser = browser
+
+    def tearDown(self) -> None:
+        self.browser.quit()
+
+    def login(self, username, password):
+        self.browser.find_element_by_id('txtUsername').send_keys(username) if username else None
+
+        if password:
+            self.browser.find_element_by_id('txtPassword').send_keys(password)
+
+        self.browser.find_element_by_id('btnLogin').click()
+
+    def enter_employee_ID(self, id):
+        self.browser.find_element_by_id('empsearch_id').send_keys(id)
+
+    def enter_employee_name(self, name):
+        el = self.browser.find_element_by_id('empsearch_employee_name_empName')
+        el.send_keys(name)
+        el.send_keys(Keys.ESCAPE)
+
+    def test_search_by_employee_ID(self):
+        self.login('admin', 'password')
+        time.sleep(1)
+        self.enter_employee_ID("441915")
+
+        self.browser.find_element_by_id('searchBtn').click()
+
+        time.sleep(1)
+
+        result = self.browser.find_element_by_xpath('//tbody/tr[1]/td[2]').text
+        self.assertEqual('441915', result)
+
+        list_of_web_elements_IDs = self.browser.find_elements_by_xpath('//*[@id="resultTable"]/tbody/tr/td[5]')
```

Comment:
You are finding the fifth column... why?
IDs are in column 2... or you can just count rows, don’t need to include td at all.

Reply:
Got it, it should have been second column (copy/paste error). I understand now I could use the //*[@id="resultTable"]/tbody/tr

Thank you for review.